docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+version: "3.7"
+services:
+
+  # expose client to internet
+  client:
+    build: .
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.client-${COMPOSE_PROJECT_NAME}.rule=Host(`client.${BASE_DOMAIN:-asbi-test.cirg.uw.edu}`)"
```

Comment:
This is pretty much the URL pattern (`$DC_SERVICE.$BASE_DOMAIN`) I use by default everywhere. If this pattern were the default router(?) in our infra traefik config, I could remove this line from every YAML file.

@jhowe-uw is that something you would be able to do in the infra traefik config? LMK if it'd be helpful to have a Cosri PivotalTracker story

Reply:
This is now captured in the below PT story:
[PT#181796672](https://www.pivotaltracker.com/story/show/181796672)